Resources/Prototypes/_Goobstation/Balloons/balloons.yml

```
@@ -0,0 +1,376 @@
+  - type: Item
+    size: Huge
+  - type: Damageable
+    damageContainer: RubberBalloon # default damage container
+  - type: Physics # has dynamic physics
+    bodyType: Dynamic
+  - type: Penetratable # projectile Able to pierce balloons
+    penetrateDamage: 1
+    damagePenalty: 0.5
+  - type: NpcFactionMember
+    factions:
+      - Balloon
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)